docs/rackhd/workflow_progress_notification.rst

```diff
@@ -16,7 +16,7 @@ RackHD will publish a workflow progress message if any of below events happens:
 
 * Progress timer timeout for an active long-run task.
 
-  Some tasks doesn't have milestones but progress information is continuous and can be got all the time. In this case a repeating timer with fixed interval is set. Progress message will be retrieved and sent each time the timer is timeout.
+  Some tasks don't have milestones but progress information is continuous and can be got all the time. In this case a repeating timer with fixed interval is set. Progress message will be retrieved and sent each time the timer is timeout.
```

Comment:
I suppose users don't need to know our internal timer. So the sentence "in this case ..." might not be necessary.